dcmgr/lib/sinatra/vnet_webapi.rb

```diff
@@ -15,9 +15,14 @@ def enable_vnet_webapi
         return if not request.request_method == "POST"
 
         if request.path_info == "/networks"
-          VNetAPIClient::Network.create(display_name: "test",
-                            ipv4_network: "192.168.100.0",
-                            ipv4_prefix: 24)
+          uuid = self.response.body.first.scan(/nw-.*?"/).uniq.first.gsub(/"/,"")
```

Comment:
Is it guaranteed for the `nw-` string to not appear anywhere else? I think this way of getting the uuid isn't safe.

I think it is safer to parse the api output as JSON or YAML and then get the uuid from there. It'll have to work for both outputs though. Still doesn't feel like the ideal solution though, since you have to check if the output is set to JSON or YAML.

Another way would be to keep the uuid in an instance variable in the networks API route but adding code there to accomodate a plugin isn't ideal either.